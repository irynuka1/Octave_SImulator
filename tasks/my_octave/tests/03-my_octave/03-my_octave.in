D 3
P 0
L 7 8
7870 7989 9289 -4430 741 4169 -3695 -9984
-5149 -6894 -1668 1278 8406 831 6029 2183
-1344 -6456 -3227 4456 9663 8838 5781 -770
-7750 -6862 -7087 4299 7361 -8127 4694 5819
9574 4859 -112 2149 -8170 -3542 -9933 401
4233 -8284 3931 -4457 -1281 -2539 3514 -3576
2973 6354 -7490 -7509 7124 -8822 -1075 8970
P 0
T 0
T 0
F 0
L 8 7
9630 435 -608 5533 4569 6924 3982
2429 9517 -4492 3617 1184 9269 3350
9549 2708 -7540 -7000 4127 -9032 -6077
-3764 -1008 -3825 6506 -9406 -2803 3476
-3144 8866 1678 4495 -3840 -9698 2580
-2399 3798 -6836 347 2890 9216 1287
2440 -7830 184 -5411 2113 9071 3960
4044 7928 2053 -6362 -5823 887 -1955
L 8 10
2496 -1412 7306 -3412 4942 1452 -1810 8576 -9645 6389
-688 -6816 7922 -67 -3837 4369 -4475 5580 2321 2637
-480 -1614 -1619 8097 7281 1197 -850 5835 3383 -1464
5277 -9922 -2577 851 5612 7614 845 -8547 2129 8954
6651 -3082 -9769 -5143 -4304 -9515 -6542 -2271 -1180 -3624
5372 9817 -1216 9868 4686 -6268 5396 1614 -5655 -6005
5716 -338 -7640 -6726 8955 -5055 -4303 5654 8032 -430
2544 804 -9011 -1329 8202 700 -5787 5344 -9175 765
T 0
M 0 1
D 2
P 2
Q