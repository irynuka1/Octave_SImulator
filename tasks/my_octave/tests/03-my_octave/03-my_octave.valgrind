==12496== Memcheck, a memory error detector
==12496== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12496== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12496== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12496== Parent PID: 12495
==12496== 
==12496== 
==12496== HEAP SUMMARY:
==12496==     in use at exit: 0 bytes in 0 blocks
==12496==   total heap usage: 96 allocs, 96 frees, 11,320 bytes allocated
==12496== 
==12496== All heap blocks were freed -- no leaks are possible
==12496== 
==12496== For lists of detected and suppressed errors, rerun with: -s
==12496== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
