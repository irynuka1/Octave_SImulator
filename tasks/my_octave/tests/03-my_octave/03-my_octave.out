No matrix with the given index
No matrix with the given index
7870 7989 9289 -4430 741 4169 -3695 -9984 
-5149 -6894 -1668 1278 8406 831 6029 2183 
-1344 -6456 -3227 4456 9663 8838 5781 -770 
-7750 -6862 -7087 4299 7361 -8127 4694 5819 
9574 4859 -112 2149 -8170 -3542 -9933 401 
4233 -8284 3931 -4457 -1281 -2539 3514 -3576 
2973 6354 -7490 -7509 7124 -8822 -1075 8970 
7 10
4132 9934 4513 6877 2522 4864 9879 9179 1913 9427 
2313 7773 5053 9066 1074 21 3617 7971 8783 1266 
3547 7367 3608 4705 5241 3439 7358 5406 9359 1863 
6618 5752 6902 4122 6751 8520 854 3770 6350 2588 
264 9097 3479 1376 3203 1403 8690 7422 4384 5519 
4808 3514 3417 1549 9810 8778 1517 497 6130 2720 
9869 1772 3333 5211 1713 8266 4815 7242 5392 1264 
